### ISSUE TITLE
Nested routes search for an incorrect relative chunk path (SSR)

### ISSUE THREAD

**Issue by njjkgeerts** [smooth-code/loadable-components#136]
## 🐛 Bug Report

First, thanks for writing this library. It seems nested paths search for an incorrect relative chunk path with server-side rendering.

Example in my project:

In `http://localhost:3000/work/viewbook`, loadable-components adds "loadable-pages-WorkShow.bundle.js" in a script tag. However, it should be "/loadable-pages-WorkShow.bundle.js", because the dist directory is mounted on '/' (root). In the 'work' directory, the chunk will never be found.

The non-nested paths (home, work, contact) work correctly with SSR and code splitting, because those paths are already at the root.

Of course, it could also be a configuration problem on my part? I tried several things, like adding the '/' to the paths after loading the manifest, but that only works in the first render (server-side).

## To Reproduce

Please see my starter project: [https://github.com/njjkgeerts/ultimate-starter-kit]

## Expected behavior

It should look for the chunks in the root '/' path. Or perhaps, one should be able to configure the public path of the chunks?

## Link to repl or repo (highly encouraged)

[https://github.com/njjkgeerts/ultimate-starter-kit]

## Run `npx envinfo --system --binaries --npmPackages @loadable/component,@loadable/server,@loadable/webpack-plugin,@loadable/babel-plugin --markdown --clipboard`

Paste the results here:

```bash
## System:
 - OS: macOS 10.14
 - CPU: x64 Intel(R) Core(TM) i7-4650U CPU @ 1.70GHz
 - Memory: 533.60 MB / 8.00 GB
 - Shell: 5.6.2 - /usr/local/bin/zsh
## Binaries:
 - Node: 8.12.0 - /usr/local/opt/node@8/bin/node
 - Yarn: 1.9.4 - /usr/local/bin/yarn
 - npm: 6.4.1 - /usr/local/opt/node@8/bin/npm
 - Watchman: 4.9.0 - /usr/local/bin/watchman
## npmPackages:
 - @loadable/babel-plugin: ^4.0.5 => 4.0.5 
 - @loadable/component: ^4.0.2 => 4.0.2 
 - @loadable/server: ^4.0.5 => 4.0.5 
 - @loadable/webpack-plugin: ^4.0.5 => 4.0.5 
```

**Comment by njjkgeerts** [comment]
So, it seems I can easily fix it by configuring Webpack with `publicPath: '/'`. 

Should this issue remain open, or is '' as the default choice for the public path desired?

**Comment by njjkgeerts** [comment]
Ah, so I can fix it by doing the following two things:
* Add '/' to the chunk paths after loading it from the manifest.
* Add publicPath: '/' to the config of Webpack.

I'm not sure why the second action is necessary... I do think the bug is easily fixed, by modiying the default publicPath in `loadable-components/packages/webpack-plugin/src/index.js` to `(publicPath || '/'`.

**Comment by njjkgeerts** [comment]
It works great now, thanks! I see you optimised the chunks even further with vendor and overlapping code. Really nice!

I see 5.1.0 is not yet released on npmjs.com, is that intentional?

**Comment by neoziro** [comment]
@njjkgeerts yeah, all is up to date. The v5.1.0 affects only the server part.
